code/game/objects/items/devices/traitordevices.dm

```diff
@@ -276,16 +304,28 @@ effective or pretty fucking useless.
 	new /obj/item/wirecutters(src)
 
 /obj/item/storage/toolbox/emergency/turret/attackby(obj/item/attacking_item, mob/living/user, params)
-	if(attacking_item.tool_behaviour == TOOL_WRENCH && user.combat_mode && attacking_item.use_tool(src, user, 2 SECONDS, volume = 50))
-		user.visible_message(span_danger("[user] bashes [src] with [attacking_item]!"), \
-			span_danger("You bash [src] with [attacking_item]!"), null, COMBAT_MESSAGE_RANGE)
-		playsound(src, "sound/items/drill_use.ogg", 80, TRUE, -1)
-		var/obj/machinery/porta_turret/syndicate/toolbox/turret = new(get_turf(loc))
-		set_faction(turret, user)
-		turret.toolbox = src
-		forceMove(turret)
+	if(!istype(attacking_item, /obj/item/wrench/combat))
+		return ..()
+
+	if(!user.combat_mode)
+		return
+
+	if(!attacking_item.toolspeed)
 		return
-	return ..()
+
+	balloon_alert(user, "constructing...")
+	if(!attacking_item.use_tool(src, user, 5 SECONDS, volume = 20))
```

Comment:
whyd you change this from 5 seconds to 2 seconds

Reply:
Wait i changed that? Huh i did not noticed it as i was coping from other so give me a second gonna change it back to 2s

Reply:
Changed back into 2s